### PR TITLE
[Exporter.Geneva] Add changelog for OTLP protobuf metric exporter.

### PR DESCRIPTION
**Pull request by vishweshbankwar**
Towards #1585 

## Changes

Adding changelog with relevant PRs.

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,13 @@
 * Update OpenTelemetry SDK version to `1.8.1`.
   ([#1668](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1668))
 
+* Add OTLP protobuf metric exporter for windows environment. Use
+  `"PrivatePreviewOtlpProtobufMetricExporter=true"` in the connection string to opt-in.
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:13]: Why is this called `PrivatePreviewOtlpProtobufMetricExporter`? I'm concerned that `OtlpProtobufMetricExporter` is going to create lots of confusion.

Comment (vishweshbankwar) [reply src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md]: @reyang - This was a typo. I had changed the name to `PrivatePreviewEnableOtlpProtobufEncoding` in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1667. Sorry for the confusion. Fixed it here as well.

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 2]
```diff
@@ -2,16 +2,24 @@
...
 * Update OpenTelemetry SDK version to `1.8.1`.
   ([#1668](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1668))
 
+* Add OTLP protobuf encoding support for metric exporter in Windows
+  environment. Use `PrivatePreviewEnableOtlpProtobufEncoding=true` in the
+  connection string to opt-in.
+  ([#1596](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1596))
+  ([#1626](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1626))
+  ([#1629](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1629))
+  ([#1634](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1634))
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:18]: I don't know what's the intended format here, maybe:

```suggestion
  ([#1596](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1596),
   [#1626](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1626),
   [#1629](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1629),
   [#1634](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1634))
```

Comment (vishweshbankwar) [reply src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:18]: This is much clear. Thanks!

File: test/OpenTelemetry.Exporter.Geneva.Tests/GenevaMetricExporterTests.cs [hunk 3]
```diff
@@ -303,12 +303,18 @@ public void DisableMetricNameValidationTest(bool disableMetricNameValidation, bo
                 {
                     if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                     {
-                        options.ConnectionString = $"Account=OTelMonitoringAccount;Namespace=OTelMetricNamespace;DisableMetricNameValidation={disableMetricNameValidation};PrivatePreviewOtlpProtobufMetricExporter={enableOtlpProtobufexporter}";
+                        options.ConnectionString = $"Account=OTelMonitoringAccount;Namespace=OTelMetricNamespace;DisableMetricNameValidation={disableMetricNameValidation}";
```
Comment (vishweshbankwar) [inline test/OpenTelemetry.Exporter.Geneva.Tests/GenevaMetricExporterTests.cs:306]: @utpilla - FYI
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1669#discussion_r1573030445

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 4]
```diff
@@ -2,16 +2,24 @@
...
-
 * Fix a bug in `GenevaMetricExporter` where the `MetricEtwDataTransport` singleton
   is disposed.
   ([#1537](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1537))
 
 * Update OpenTelemetry SDK version to `1.8.1`.
   ([#1668](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1668))
 
+* Add OTLP protobuf encoding support for metric exporter in Windows
+  environment. Use `PrivatePreviewEnableOtlpProtobufEncoding=true` in the
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:13]: Cover this in the doc https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Exporter.Geneva#genevametricexporteroptions-for-metrics.

File: src/OpenTelemetry.Exporter.Geneva/README.md [hunk 5]
```diff
@@ -255,6 +255,17 @@ On Linux provide an `Endpoint` in addition to the `Account` and `Namespace`.
...
 
+`PrivatePreviewEnableOtlpProtobufEncoding`: Set the key value to `true` to opt-in.
+
+`PrivatePreviewEnableOtlpProtobufEncoding` is a temporary key offered to change
+the underlying serialization format to OTLP binary protobuf following the schema
+defined in
+[specification](https://github.com/open-telemetry/opentelemetry-proto/blob/v1.1.0/opentelemetry/proto/metrics/v1/metrics.proto).
+
+> [!NOTE]
+ > Exporting [Exemplar](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#exemplar) and [ExponentialHistogram](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#exponentialhistogram) is not supported when enabling PrivatePreviewEnableOtlpProtobufEncoding.
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Geneva/README.md:266]: not supported -> not supported for now, add links to the tracking issues.

File: src/OpenTelemetry.Exporter.Geneva/README.md [hunk 6]
```diff
@@ -255,6 +255,17 @@ On Linux provide an `Endpoint` in addition to the `Account` and `Namespace`.
 For example:
 `Endpoint=unix:{UDS Path};Account={MetricAccount};Namespace={MetricNamespace}`.
 
+`PrivatePreviewEnableOtlpProtobufEncoding`: Set the key value to `true` to opt-in.
+
+`PrivatePreviewEnableOtlpProtobufEncoding` is a temporary key offered to change
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Geneva/README.md]: temporary key -> experimental feature

File: src/OpenTelemetry.Exporter.Geneva/README.md (hunk 5 above)
Comment (reyang) [reply src/OpenTelemetry.Exporter.Geneva/README.md]: nit: is -> are

File: src/OpenTelemetry.Exporter.Geneva/README.md [hunk 7]
```diff
@@ -255,6 +255,19 @@ On Linux provide an `Endpoint` in addition to the `Account` and `Namespace`.
...
 `Endpoint=unix:{UDS Path};Account={MetricAccount};Namespace={MetricNamespace}`.
 
+Set `PrivatePreviewEnableOtlpProtobufEncoding=true` to change
+the underlying serialization format to binary protobuf following the schema
+defined in [OTLP
+specification](https://github.com/open-telemetry/opentelemetry-proto/blob/v1.1.0/opentelemetry/proto/metrics/v1/metrics.proto).
+
+> [!NOTE]
+ > `PrivatePreviewEnableOtlpProtobufEncoding` opt-in setting is currently
+ > only supported on Windows environment. Exporting
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Geneva/README.md:265]:
```suggestion
 > only supported in Windows environment. Exporting
```

**Review by reyang — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Geneva/README.md (hunk 5 above)
Comment (vishweshbankwar) [reply src/OpenTelemetry.Exporter.Geneva/README.md]: done

**Review by CodeBlanch — APPROVED** [review]
LGTM

**Pull request merged**